parsl/tests/test_htex/test_htex.py

```diff
@@ -44,3 +65,42 @@ def test_htex_start_encrypted(
         assert htex.outgoing_q.zmq_context.cert_dir is None
         assert htex.incoming_q.zmq_context.cert_dir is None
         assert htex.command_client.zmq_context.cert_dir is None
+
+
+@pytest.mark.local
+@pytest.mark.parametrize("started", (True, False))
+@pytest.mark.parametrize("timeout_expires", (True, False))
+@mock.patch(f"{_MOCK_BASE}.logger")
+def test_htex_shutdown(
+    mock_logger: mock.MagicMock,
+    started: bool,
+    timeout_expires: bool,
+    htex: HighThroughputExecutor,
+):
+    mock_ix_proc = mock.MagicMock()
+
```

Comment:
As a matter of habit, I suggest we use `spec=` where feasible.  In this case, we know the object this is supposed to mock, so consider:
```python
mock_ix_proc = mock.Mock(spec=ForkProcess)
```